### DISCUSSION TITLE
(SPANISH) pregunta para la carta "birthday.TIC"

### DISCUSSION THREAD

**Discussion by eggbanana173** [nesbox/TIC-80#1677] [Q&A]
hola nesbox (no te molestare pidiéndote ayuda por ahora) te quería hacer una pregunta
¿puedo poner a la mascota "Ticsy" en "birthday.TIC" ?
claro que escribiré en los créditos (o metadatos) que eres el autor original de TICSY

>-- title:  birthday
-- author: mobius line (mi cuenta en tic 80)
-- desc:   a virtual birthday if you are alone
-- script: lua
-- thanks NESBOX for TIC 80 and TICSY

**Comment by nesbox** [comment]
I think yes
